packages/mintlify-cli/src/index.ts

```diff
@@ -4,12 +4,16 @@
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 import { dev, installDepsCommand } from "@mintlify/previewing";
+import {
+  getBrokenInternalLinks,
+  renameFileAndUpdateLinksInContent,
+} from "@mintlify/link-rot";
 
 export const cli = () =>
   yargs(hideBin(process.argv))
     .command(
       "dev",
-      "Runs Mintlify locally (Must run in directory with mint.json)",
+      "Run Mintlify locally",
       () => {},
       async (argv) => {
         await dev(argv);
```

Comment:
I think this could be simplified to:
```suggestion
      dev
```
Beed to remove `}` manually, I was not able to insert a comment at line 20.